test/common/http/header_map_impl_test.cc

```diff
@@ -14,11 +14,6 @@ using ::testing::InSequence;
 namespace Envoy {
 namespace Http {
 
-class VerifiedHeaderMapImpl : public HeaderMapImpl {
```

Comment:
how come this is no longer needed?

Reply:
It is in this change. Good catch this should have been reverted in my branch. Will revert.